src/libraries/System.Net.Http/src/System/Net/Http/BrowserHttpHandler/HttpClientHandler.Browser.cs

```diff
@@ -0,0 +1,168 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Net.Security;
+using System.Runtime.Versioning;
+using System.Security.Authentication;
+using System.Security.Cryptography.X509Certificates;
+using System.Threading;
+using HttpHandlerType = System.Net.Http.BrowserHttpHandler;
```

Comment:
Do we still need the alias if the file is only for browsers?

Reply:
Yes because the instance variable using this type is shared and initiated differently in the constructor

Reply:
Actually later i moved the variable too, so yes this is not needed anymore